test/testcafe/spec/IdentifyRecovery_spec.js

```diff
@@ -52,6 +52,7 @@ test.requestHooks(identifyRecoveryMock)('should have correct display texts', asy
   await t.expect(saveButtonText).eql('Next');
 
   await t.expect(await identityPage.signoutLinkExists()).ok();
+  await t.expect(await identityPage.helpLinkExists()).notOk();
```

Comment:
nit: imo the `ok/notOk` assertions are not very precise because they only check truthy/falsy values. better assertions would be `.eql(true)` or `.eql(false)`

Reply:
`exists` from TestCafe Selector API returns only boolean, so should not be a problem in this case